modules/ROOT/pages/_partials/mobile-client-registration-procedure.adoc

```diff
@@ -13,5 +13,7 @@ image:catalog-all.png[Available Mobile Clients]
 . Click the *Apps* tab and then choose the mobile platform (Android, iOS, Cordova or Xamarin) and follow the wizard.
 +
 NOTE: On the wizard's *configuration* screen, it is recommended that you input the name of the showcase application into the *Package Name* text box.
++
```

Comment:
what is the motivation behind this?

Reply:
aha, i presume this is the culprit https://github.com/aerogear/mobile-docs/pull/453/commits/8bee7ce903d3c66856987f24aee89b042ad8bfe0